src/overlays/actors/ovl_Bg_Ingate/z_bg_ingate.c

```diff
@@ -188,7 +190,7 @@ void func_80953F9C(BgIngate* this, GlobalContext* globalCtx) {
 
         if (this->unk160 & 2) {
 
-            if (this->unk164->unk1 != 0xFF) {
+            if (this->timePath->unk1 != 0xFF) {
```

Comment:
Agreed, though should probably be done in a separate PR (along with count? and unk2). I had planned to change this in an early `SubS`, but decided not to for a couple reasons.
1. It would be a large diff and distract from the real purpose of the PR
2. It would cause a lot of conflicts (not as much a worry anymore with less PRs) especially since it would be nearly impossible to use namefixer for it.